src/clr-addons/export-datagrid-button/export-datagrid-button.component.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, ElementRef, Input, Output, EventEmitter } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+import zipcelx, { ZipCelXConfig } from 'zipcelx';
+
+export enum ExportTypeEnum {
+  ALL = 'ALL',
+  FILTERED = 'FILTERED',
+  SELECTED = 'SELECTED',
+}
+
+interface ExportType {
+  type: ExportTypeEnum;
+  title: string;
+}
+
+@Component({
+  selector: 'clr-export-datagrid-button',
+  templateUrl: './export-datagrid-button.component.html',
+  styleUrl: './export-datagrid-button.component.scss',
+  standalone: false,
+})
```

Comment:
can you make this standalone and use input signals? that should be the default at this point